frontend/src/pages/productpage/ProductPage.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react'
+import { Productdb } from '../../api/db/Productdb'
+import Product from '../../components/product/Product'
+function ProductPage() {
+  const [products] = useState(Productdb)
```

Comment:
Can you please explain what are you doing here?
Why put `Productdb` in a state and that too when you're not using the state's setter function?

You can directly use Productdb and map over it (as it is an array of objects)

Reply:
ok I will do that , thanks